tfhe/benches/utilities.rs

```diff
@@ -357,6 +535,9 @@ pub fn write_to_json<
     fs::write(params_directory, serde_json::to_string(&record).unwrap()).unwrap();
 }
 
+#[allow(dead_code)]
```

Comment:
Here a cfg("feature = gpu") would be better no?

Reply:
It makes sense to add it yes, but I still have to put `allow(dead_code)` macro since all gpu benchmarks target won't use this constant.

Reply:
In that case you could use `#cfg[not(feature = "gpu"))]` maybe?